Lib/test/test_print.py

```diff
@@ -128,5 +128,26 @@ def flush(self):
                 raise RuntimeError
         self.assertRaises(RuntimeError, print, 1, file=noflush(), flush=True)
 
+
+class TestPy2MigrationHint(unittest.TestCase):
+    """Test that correct hint is produced analogous to Python3 syntax,
+    if print statement is executed as in Python 2.
+    """
+
+    def test_normal_string(self):
+        python2_print_str = 'print "Hello World"'
+        with self.assertRaises(SyntaxError) as context:
+            exec(python2_print_str)
+
+        self.assertTrue('print("Hello World")' in str(context.exception))
```

Comment:
Use `assertIn()`.

Or use `assertRaisesRegex()` with the argument `r'print\("Hello World"\)'` or `re.escape('print("Hello World")')`.